### PR TITLE
Update Wesley documentation

### LINKED ISSUE #20: Wesely documentation
https://github.com/EarthSciML/AtmosphericDeposition.jl/blob/0fae86b85024981b930c473aa59434c1805f1ac3/docs/src/wesley1989.md?plain=1#L1

On this page, add an overview and some examples, similiar to [here](https://github.com/EarthSciML/Aerosol.jl/blob/main/docs/src/isorropia/overview.md?plain=1) and [here](https://github.com/EarthSciML/Aerosol.jl/blob/main/docs/src/isorropia/examples.md?plain=1) except maybe it can all go on one page.

The examples can maybe some plots that show how varying different parameters influences the deposition rate.

### PR DESCRIPTION
**Pull request by jialinl6**
Fixes #20 
Captures https://github.com/EarthSciML/AtmosphericDeposition.jl/issues/20
Claims https://github.com/EarthSciML/AtmosphericDeposition.jl/issues/20

### CODE REVIEW

**Comment by ctessum** [comment]
## <a href="https://gov4git.org">This project is governed with Gov4Git.</a> Gov4Git notices
On `Thursday, 29-Feb-24 21:47:31 UTC` by Gov4Git `dev`

#### Notice `yjg5h7`
Started managing this PR as Gov4Git proposal `34` with initial __approval score__ of `0.000000`.

This project is managed by [Gov4Git](https://gov4git.org), a decentralized governance system for collaborative git projects.
To participate in governance, __install the [Gov4Git desktop app](https://github.com/gov4git/desktop-application)__.

**Review by ctessum — APPROVED** [review]

File: docs/src/wesley1989.md [hunk 1]
```diff
@@ -1,30 +1,69 @@
...
+
+## Main functions
+Function ```WesleySurfaceResistance``` is used to calculate surface resistance to dry depostion [s/m] based on Wesely (1989) equation 2.
+The inputs of the function are information on the chemical of interest ```gasData```, solar irradiation ```G``` [W/m²], the surface air temperature ```Ts``` [°C], the slope of the local terrain ```Θ``` [radians], the season index ```iSeason```, the land use index ```iLandUse```, whether there is currently rain or dew ```rain``` or ```dew```, and whether the chemical of interest is either SO2 ```isSO2``` or O3 ```isO3```.
+Here's an example:
+
+```julia @example1
```
Comment (ctessum) [inline docs/src/wesley1989.md:23]: I think this should probably be `julia @example 1` instead of `julia @example1`

**Comment by ctessum** [comment]
## <a href="https://gov4git.org">This project is governed with Gov4Git.</a> Gov4Git notices
On `Friday, 01-Mar-24 01:40:08 UTC` by Gov4Git `dev`

#### Notice `rxvsq2`
This PR's __approval score__ is now `3.000000`.
The __cost of review__ is `9.000000`.
The __projected bounty__ is now `0.000000`.

#### Notice `arvvq2`
The set of eligible issues claimed by this PR is empty.

File: docs/src/wesley1989.md (hunk 1 above)
Comment (jialinl6) [reply docs/src/wesley1989.md:23]: Oh, I see! I will correct this!

**Comment by ctessum** [comment]
## <a href="https://gov4git.org">This project is governed with Gov4Git.</a> Gov4Git notices
On `Friday, 01-Mar-24 03:13:18 UTC` by Gov4Git `dev`

#### Notice `yvppic`
This PR referenced https://github.com/EarthSciML/AtmosphericDeposition.jl/issues/20, managed as Gov4Git concern `20`.

**Review by ctessum — APPROVED** [review]

**Pull request merged**